arangod/GeneralServer/SslServerFeature.cpp

```diff
@@ -592,3 +597,90 @@ std::string SslServerFeature::stringifySslOptions(uint64_t opts) const {
   // strip initial comma
   return result.substr(2);
 }
+
+
+static void splitPEM(std::string const& pem, std::vector<std::string>& certs, std::vector<std::string>& keys) {
+  std::vector<std::string> result;
```

Comment:
are PEM files always in this format ?

Reply:
As far as I know, yes. There might be people who use some other file and call it .pem, but everything I have seen so far and the definition of PEM is this.